src/99-appendices/04-entity-table.md

```diff
@@ -7,18 +7,26 @@ precede the reconstruction entity. REQUIRED and OPTIONAL entities for a given
+| dwi<br>(dwi bvec bval)                                                                         | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                | OPTIONAL                 | OPTIONAL      |                        |                |                     |                       |                 |
+| fmap<br>(phasediff phase1 phase2 magnitude1 magnitude2 magnitude fieldmap)                     | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |
+| fmap<br>(epi)                                                                                  | REQUIRED      | OPTIONAL      |                | OPTIONAL      | OPTIONAL                 |                | REQUIRED                 | OPTIONAL      |                        |                |                     |                       |                 |
+| beh<br>(events stim physio)                                                                    | REQUIRED      | OPTIONAL      | REQUIRED       |               |                          |                |                          |               |                        |                |                     |                       |                 |
+| meg<br>                                                                                        | REQUIRED      | OPTIONAL      | REQUIRED       | OPTIONAL      |                          |                |                          | OPTIONAL      |                        |                |                     | OPTIONAL              |                 |
+| eeg<br>                                                                                        | REQUIRED      | OPTIONAL      | REQUIRED       | OPTIONAL      |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |
+| ieeg<br>                                                                                       | REQUIRED      | OPTIONAL      | REQUIRED       | OPTIONAL      |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |
+| channels<br>(meg/eeg/ieeg)                                                                     | REQUIRED      | OPTIONAL      | REQUIRED       |               |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |
```

Comment:
From here down, the format has switched from `DATATYPE (SUFFIX1 SUFFIX2 SUFFIX3)` to `SUFFIX (DATATYPE1 ...)`. I recognize that it was there before, so whether we want to fix it here or push it to another PR, I'm okay either way).

Reply:
there are more extensive changes to be done (also table Css, potentially splitting per modality, ...), so let's keep the PRs short and focused - that keeps motivation up and improvements flowing in regularly